main.cpp

```diff
@@ -455,4 +454,8 @@ int main() {
 	return 0;
 }
 
+int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine, int nCmdShow) {

```

Comment:
I guess it's better to keep the standard `int main()` if additional parameters (such as `hInstance`) are not required:
https://stackoverflow.com/a/11785733/4399840